### ISSUE TITLE
extra window  opened after binding editor to  existing console

### ISSUE THREAD

**Issue by alexey0308** [jupyterlab/jupyterlab#4062]
Thanks for this useful feature about sending lines from an editor!

If  I choose an existing kernel in 'Create a console for editor', an extra console is opened.
It is possible to send lines to the kernel from the file X1 iff console for X1 is opened,
and from the file X2 iff its console is opened. Both consoles represent the same kernel.

It would be very nice if it would be possible to keep only one window for a given kernel, to
which a user can send lines from several files.

version 0.31.8

Regards,
Alex

**Comment by jasongrout** [comment]
Good point. You can, of course, have both consoles pointing to the same kernel. However, it's not possible right now to point a file to an existing console window.

Thanks for bringing up the idea!

**Comment by alexey0308** [comment]
Great, thanks for taking it into account!
